src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userEmail = new UserService();
+        User user = userEmail.findByEmail(email);
+
+        if (user != null) {
+            if (password.equals(user.getPassword())) {
+                return true;
+            }
+        }
         return false;
```

Comment:
let's simplify your code, try to avoid such nested constructions
```suggestion
	return user != null && password.equals(user.getPassword())
```